docs/source/getstarted.rst

```diff
@@ -60,74 +61,202 @@ Let's test your new setup by running a very simple program. This program instruc
+  a. For macOS and Linux systems, type the following and press **Enter**::
 
-        cd tutorials/01_basics
+    cd tutorials/01_basics
 
-    b. For Windows systems, type the following and press **Enter**::
+  b. For Windows systems, type the following and press **Enter**::
 
-        cd tutorials\01_basics
+    cd tutorials\01_basics
 
 Then, run the program.
 
-    a. For macOS and Linux systems, type the following and press **Enter**::
+  a. For macOS and Linux systems, type the following and press **Enter**::
 
-        ./01_hello_world.py
+    ./01_hello_world.py
 
-    b. For Windows systems, type the following and press **Enter**::
+   The same can also be achieved by writing
+	
+	python3 01_hello_world.py
 
-        py 01_hello_world.py
+  b. For Windows systems, type the following and press **Enter**::
+
+    py 01_hello_world.py
 
 2. If done correctly, Cozmo will say "Hello, World!"
 
 .. warning:: If Cozmo does not perform as expected, look at the first Terminal window and make sure no error messages appeared. If you continue to have issues, please seek help in the Forums.
 
 The code for the Hello World program can be `viewed here. <https://github.com/anki/cozmo-python-sdk/tree/master/examples/tutorials/01_basics/01_hello_world.py>`_
 
-We can edit this code to make Cozmo say something new. Let's write our first program using this code.
+
+You are now all set up to run python programs on Cozmo. Next we will go over how to edit the above code to make Cozmo say something new. Let's write our first program.
 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
-Next Steps - "Night-Night"
+"Night-Night"
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. Copy ``01_hello_world.py`` to a new file named ``nightnight.py``.
+1. Copy ``01_hello_world.py`` to a new file named ``nightnight.py`` by doing the following:
 
-    a. For macOS and Linux systems, type the following and press **Enter**::
+  a. For macOS and Linux systems, type the following and press **Enter**::
 
-        cp 01_hello_world.py nightnight.py
+    cp 01_hello_world.py nightnight.py
 
-    b. For Windows systems, type the following and press **Enter**::
+  b. For Windows systems, type the following and press **Enter**::
 
-        copy 01_hello_world.py nightnight.py
-    
+    copy 01_hello_world.py nightnight.py
+
+Now, nightnight.py is saved in the same folder as 01_hello_world.py.
+  
 2. Open this new document in a source code editor or plain-text editor. Free source code editors, such as `PyCharm Community Edition <https://www.jetbrains.com/pycharm/>`_ , `Atom <https://atom.io>`_ , `Sublime <https://www.sublimetext.com>`_ , or `TextWrangler <http://www.barebones.com/products/textwrangler/>`_ can be found online. Anki does not provide tech support for third-party source code editors.
 
-3. Each line in the program relates to a specific function. To learn more, see :doc:`the Beginner's Tutorial </tutorial-beginner>`.
+3. The code for the program currently looks like this:
```

Comment:
We would like to avoid putting too much of the program code here as, if we modify the file, we will have to come back here and modify it here as well, so this should be removed.